xtext/org.icyphy.linguafranca/src/org/icyphy/generator/CGenerator.xtend

```diff
@@ -406,8 +406,50 @@ class CGenerator extends GeneratorBase {
                 initializeTriggerObjectsEnd = new StringBuilder()                
                         
                 // Enable clock synchronization if the federate is not local and clock-sync is enabled
+//<<<<<<< HEAD
```

Comment:
This looks like a weird merge conflict. I've refactored everything to `initializeClockSynchronization()` on `master` so I think these can be safely removed.

Reply:
Jep